WebRoot/public/login.jsp

```diff
@@ -116,7 +116,7 @@
                     <c:redirect url="${logoutRedirectUrl}"/>
                 </c:when>
                 <c:when test="${touchSupported and touchLoginPageExists and (empty param.client or param.client eq 'touch') and
-                    (empty param.virtualacctdomain) and (empty virtualacctdomain)}">
+                    (empty param.virtualacctdomain) and (empty virtualacctdomain) and zimbraxSupported ne true}">
```

Comment:
`and (zimbraxSupported ne true)`
for consistancy

Reply:
Done